app/assets/images/icons/loading.svg

```diff
@@ -1,4 +1,4 @@
-<svg class="animate-spin -ml-1 mr-3 h-5 w-5" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
```

Comment:
This prevented the spin animation from being general use, but I'm not sure where and why it was there. Is it acceptable for this to be removed? I'm OOTL on where the component came from.

Reply:
I thought it was being used in the infinite scroll / Pagy component.